main/tests/server/src/com/google/refine/expr/functions/CrossTests.java

```diff
@@ -64,6 +64,8 @@ public void init() {
         logger = LoggerFactory.getLogger(this.getClass());
     }
 
+    private HasFieldsListImpl rows;
```

Comment:
```suggestion
    private HasFieldsListImpl emptyList;
```
I would rather call this `emptyList` to make it clearer that it is used for test cases where we expect the return value of the function to be empty. (Of course changing this attribute name will require updating the rest of the test.

Reply:
@wetneb 
Okay, I would do that